src/Faker/Provider/Lorem.php

```diff
@@ -196,7 +196,7 @@ public static function text($maxNbChars = 200)
                 $word = ($size ? ' ' : '') . static::$type();
                 $text[] = $word;
 
-                $size += strlen($word);
+                $size += mb_strlen($word);
```

Comment:
This is a breaking change where you require an additional php extension for this.

We can do this with the polyfil but not like this

Reply:
I understand and I agree, this should not  break.  
I don't understand how the polyfil can help us as it uses `iconv` and we don't requite `ext-iconv`.  
maybe something like this?

```php
    /**
     * @param string      $string
     * @param string|null $encoding
     *
     * @return int
     */
    public static function strlen($string, $encoding = null)
    {
        if(method_exists(get_called_class(), 'provider_strlen')){
            return call_user_func([get_called_class(), 'provider_strlen'], $string, $encoding);
        }

        return strlen($string);
    }
```
Then Lorem would call 
```php
$size += static::strlen($word);
```
And my Lorem will do
```php
    public static function provider_strlen($string, $encoding = null)
    {
        return mb_strlen($string, $encoding);
    }
```

Reply:
Looking at this now, I have no idea why I reached out for this, I think simple inheritance would work fine here